models/wholeBrainSeg_Large_UNEST_segmentation/docs/wholeBrainClassesList.txt

```diff
@@ -0,0 +1,4 @@
+Third Party Licenses
```

Comment:
What's this file used for?

Thanks.

Reply:
It was a list of brain ROIs used for training, removed.